packages/embed/src/utils/build-iframe-src.ts

```diff
@@ -22,19 +34,14 @@ const mapOptionsToQueryParams = (type: EmbedType, options: UrlOptions): Record<s
 export const buildIframeSrc = (params: BuildIframeSrcOptions): string => {
   const { formId, type, embedId, options } = params
 
-  const url = new URL(`https://form.typeform.com/to/${formId}`)
+  const queryParams = mapOptionsToQueryParams(type, embedId, addDefaultUrlOptions(options))
 
-  if (embedId) {
-    url.searchParams.set('typeform-embed-id', embedId)
-  }
-
-  const queryParams = mapOptionsToQueryParams(type, options)
-  Object.entries(queryParams).forEach(([paramName, paramValue]) => {
-    if (!paramValue) {
-      return
-    }
-    url.searchParams.set(paramName, paramValue)
-  })
+  const url = new URL(`https://form.typeform.com/to/${formId}`)
+  Object.entries(queryParams)
+    .filter(([_paramName, paramValue]) => paramValue !== undefined && paramValue !== null)
```

Comment:
This is to avoid removing zero value which is still a valid value and we want to pass it to query params but it evaluates as falsy.

Reply:
As a fan of short syntax, I must suggest the following:

```suggestion
    .filter(([, paramValue]) => paramValue != null)
```

Reply:
You can do !! to check for a falsy value, also, why not just do a non-strict check for null which typescript should handle for both undefined and null?
Also, I'd consider removing the filter and doing the check in the forEach since the entries are not reused as that allocates extra memory
```typescript
Object.entries(queryParams).forEach([key, value] => {
  if (!!value) {
    url.searchParams.set(paramName, paramValue)
  }
})
```

Reply:
`!!value` prevents us from adding empty strings or zeros which is incorrect.